src/Command/StateMachineInitCommand.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the License Agreement. See LICENSE file.
+ */
+
+namespace StateMachine\Command;
+
+use Cake\Command\Command;
+use Cake\Console\Arguments;
+use Cake\Console\ConsoleIo;
+use Cake\Core\Configure;
+use Cake\Utility\Inflector;
+use StateMachine\StateMachineConfig;
+
+/**
+ * Command class for initializing.
+ */
+class StateMachineInitCommand extends Command
+{
+    /**
+     * @inheritDoc
+     */
+    public static function defaultName(): string
+    {
+        return 'state_machine init';
+    }
+
+    /**
+     * @param \Cake\Console\Arguments $args
+     * @param \Cake\Console\ConsoleIo $io
+     *
+     * @return int|null|void
+     */
+    public function execute(Arguments $args, ConsoleIo $io)
+    {
+        $stateMachine = (string)$args->getArgumentAt(0);
+        $stateMachine = Inflector::camelize($stateMachine);
+
+        $io->out('Initialize state machine ' . $stateMachine . ':');
+        $processName = $stateMachine . '01';
+        $config = new StateMachineConfig();
+        $pathToXml = $config->getPathToStateMachineXmlFiles() . $stateMachine . DS;
+        $filePath = $pathToXml . $processName . '.xml';
+        if (!$args->getOption('overwrite') && file_exists($filePath)) {
```

Comment:
I would move such things to separate methods for the readability sake.
E.g. this check can go to some assert method, xml file creation to another one. It will keep the main public method clean and easy to navigate. Smth like a table of content